documentation/sphinx/source/architecture.rst

```diff
@@ -3,7 +3,7 @@ Architecture
 ############
 
 FoundationDB makes your architecture flexible and easy to operate. Your applications can send their data directly to the FoundationDB or to a :doc:`layer<layer-concept>`, a user-written module that can provide a new data model, compatibility with existing systems, or even serve as an entire framework. In both cases, all data is stored in a single place via an ordered, transactional key-value API.
-  
+
 The following diagram details the logical architecture.
 
-.. image:: /images/Architecture.pdf
\ No newline at end of file
+.. image:: /images/Architecture.png
```

Comment:
Is the PNG also built by Sphinx? Just asking because the current `images` folder doesn't contain a PNG file:

https://github.com/apple/foundationdb/tree/master/documentation/sphinx/source/images

Reply:
no the png was extracted from the architecture.pdf and provided by @kjc-stripe